docs/command-reference.md

```diff
@@ -62,6 +62,7 @@ Downloads container tool (Containerd, BuildKit, nerdctl) asynchronously and inst
 | DownloadPath | String | Path to download container tools  | `$HOME\Downloads` |
 | Cleanup | Switch | Specifies whether to cleanup after installation is done  |  |
 | Force | Switch | Force install the tools even if they already exists at the specified path |  |
+| Setup | Switch | Register and Start Conatinerd and Buildkitd services |  |
```

Comment:
nit: I feel like the name `Setup` is too generic, perhaps something more idiomatic like `RegisterService` would be better?

Reply:
Though also thinking about it, could we just have more of opt-out than opt-in? That is, by default, register the service but if someone wants to opt-out, they can provide the opt-out switch like `SkipServiceRegistration` or something.

Reply:
@billyoo I had initially proposed `Register-Service`/`Register-Services` as the switch. I think we can make that change then make a decision on what the default should be after.

Reply:
@billyoo `Register and Start Conatinerd and Buildkitd services and set up NAT network`

Reply:
@iankingori `RegisterService`. The hyphen is not a vaid param name.